step.yml

```diff
@@ -0,0 +1,76 @@
+title: |-
+  Export APK from AAB
+summary: |
+  Fancy summary
+description: |
+  Fancy description
+website: https://github.com/bitrise-steplib/bitrise-step-generate-universal-apk
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-generate-universal-apk
+support_url: https://github.com/bitrise-steplib/bitrise-step-generate-universal-apk/issues
+host_os_tags:
+  - osx-10.10
+  - ubuntu-16.04
+project_type_tags:
+  - android
+type_tags:
+  - test
+
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/bitrise-step-generate-universal-apk
+
+
+inputs:
+  - aab_path:
+    opts:
+      title: "The path of the AAB."
+      summary: Summary. No more than 2-3 sentences.
+      description: |
+        Description of this input.
+
+        Can be Markdown formatted text.
+      is_expand: true
+      is_required: true
+  - keystore_path:
```

Comment:
Pls use the same Keystore related inputs as the sign-apk step: https://github.com/bitrise-steplib/steps-sign-apk/blob/master/step.yml#L96-L126